nixos/modules/programs/bash/bash.nix

```diff
@@ -78,10 +78,10 @@ in
       promptInit = mkOption {
         default = ''
           # Provide a nice prompt if the terminal supports it.
-          if [ "$TERM" != "dumb" -o -n "$INSIDE_EMACS" ]; then
+          if [ "$TERM" != "dumb" ] || [ -n "$INSIDE_EMACS" ]; then
```

Comment:
Do you think it's too different to switch from `[ .. -o .. ]` to `[[ .. || .. ]]` ?

Reply:
While `[ … -o … ]` is fundamentally unreliable due to a [grammatical ambiguity](https://rg1-teaching.mpi-inf.mpg.de/unixffb-ss98/quoting-guide.html#ssect:appl-test), I don’t have a preference between `[ … ] || [ … ]` and `[[ … || … ]]` except that the latter is non-portable (which isn’t a concern here), so I went with the one that’s more common.

Reply:
Makes sense.